lib/common/disabled_button.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+import './format_text.dart';
+
+class DisabledButton extends StatelessWidget {
+  final String text;
+  final double minWidth;
+  final double height;
```

Comment:
This is for the "Link sent" widget! I know it's not supposed to be a button but I found that it is easier to just disable it for now so it could still resemble the other primary buttons. We can defs refactor this later, customize and get rid of the `MaterialButton` usage